tests/integration_test.py

```diff
@@ -133,7 +138,10 @@ async def test_attachment_plugin_new_record(
 
     with open("kinto-logo.svg", "rb") as attachment:
         assert requests.post(
-            f"{server}/buckets/main-workspace/collections/product-integrity/records/logo/attachment",
+            urljoin(
+                server,
+                "/buckets/main-workspace/collections/product-integrity/records/logo/attachment",
+            ),
```

Comment:
Alternatively, you could use `client.get_endpoint("record", bucket="main-workspace", collection="product-integrity", id="logo")) + "/attachment"`

Reply:
https://github.com/mozilla-services/kinto-dist/pull/1970/commits/ff29e25e902fc385330815b9b1b581a63dbaf6a1#diff-f4dc9ee1e5439f84f46e5c082c2bcc2610f2a63b24ef8513ed9a7753b482f3f3R138-R144